docs/tutorial.md

```diff
@@ -85,6 +85,16 @@ adapter:
   search_query: repo:YOURUSERNAME/shepherd-demo path:/ filename:.eslintrc
 ```
 
+**Note:** you can provide an `org` instead of `search_query`. This will return a list of every visible repo in a github organization.
```

Comment:
```suggestion
**Note:** you can provide an `org` instead of `search_query`. This will return a list of every visible repo in a GitHub organization.
```